test/test.py

```diff
@@ -263,6 +263,25 @@ def tree_diff(path1, path2):
         output = subprocess.check_output([os.path.join(checkout, "tools/tree-diff"), path1, path2])
         return json.loads(output)
 
+    @staticmethod
+    def have_kernel_fs_support(fs: str) -> bool:
+        """Check for Kernel File-System Support
+
+        Check whether the current kernel supports the given file-system. This
+        is done by checking the content of /proc/filesystems.
+        """
+        supported_fs = []
+        with open("/proc/filesystems", "r", encoding="utf-8") as fss:
+            for line in fss:
+                elements = line.split()
+                if len(elements) == 2:
+                    supported_fs.append(elements[1])
+                else:
+                    supported_fs.append(elements[0])
+        if fs == "btrfs":
+            return False
```

Comment:
This is for testing the skip rule I guess?

Reply:
🤦  I forgot to remove this from when I was testing it locally. Good catch!